peepingtom/io_/read/read.py

```diff
@@ -2,14 +2,14 @@
 from collections import defaultdict
 from itertools import zip_longest
 
-from ..utils import _path, listify
+from ..utils import _path
+from ...utils import listify
```

Comment:
why are there two utils modules?

Reply:
One only contains `io` utils and is inside the `io` directory, the other is the general one... maybe should think of another name?